.github/workflows/publish.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - uses: gradle/wrapper-validation-action@v1
-      - run: ./gradlew publish --stacktrace
+      - run: ./gradlew curseforge publish --stacktrace
         env:
-          CURSEFORGE_API_KEY: ${{ secrets.CURSEFORGE_API_KEY }}
+          CURSE_API_KEY: ${{ secrets.CURSEFORGE_API_KEY }}
```

Comment:
someone remind me to add the new api key

Reply:
bruh just add me to the org conc

Reply:
did i not already add it

Reply:
it’s fixed now